hcloud/_client.py

```diff
@@ -26,7 +27,7 @@
 from .volumes import VolumesClient
 
 
-class PollIntervalFunction(Protocol):
+class BackoffFunction(Protocol):
```

Comment:
Is this a breaking change or not part of the exported API?

Reply:
It is not exposed, so not a breaking change. But might be worth to expose once the code has settled.